Packs/Veeam/pack_metadata.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "Veeam App for Palo Alto Networks XSOAR",
```

Please change to:
```suggestion
    "name": "Veeam",
```
This should fix the reoccurring IF113 validation error for all the json files.

---

Hi @samuelFain, there is a package name approved by our management. We cannot change it.
Also, we don't want field names like this veeamappforpaloaltonetworksxsoarveeamnumberofstoredrestorepoints.
Can we ignore this validation in .pack-ignore?

---

@nikita-konoplin Fine by me.

---

done